.circleci/config.yml

```diff
@@ -108,12 +108,22 @@ jobs:
           name: Run acceptance tests
           command: './deploy-scripts/bin/acceptance_tests'
       - slack/status: *slack_status
+  smoke_tests:
+    docker: *ecr_base_image
+    steps:
+      - setup_remote_docker
+      - run: *deploy_scripts
+      - run:
+          name: Run smoke tests
+          command: './deploy-scripts/bin/smoke_tests'
+      - slack/status: *slack_status
 
 workflows:
   version: 2
   test_and_build:
     jobs:
       - test
+      - smoke_tests
```

Comment:
is this one needed?

Reply:
Sorry, I opened the PR too soon. I added the line to test it and see it working. Now that is working I removed the line 😅